### ISSUE TITLE
[Resources] Remove instagram gallery from resources.html page

### ISSUE THREAD

**Issue by jazzij** [bereacollegemakerspace/bereacollegemakerspace.github.io#23]
was put there as practice I think. is currently duplicated on main page

**Comment by Harshal662** [comment]
![image](https://github.com/bereacollegemakerspace/bereacollegemakerspace.github.io/assets/79760384/365b771d-102c-45a9-9167-b3a6aba84777)

you mean this gallery below insta account

**Comment by jazzij** [comment]
@Harshal662 Yes, this is the gallery. Can remove the gallery code from Resources page.
